angular/src/lib/alert-meeting/alert-meeting.component.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  TemplateRef,
+} from '@angular/core';
+
+import { AlertMeetingAttendee } from './alert-meeting-config';
+
+// Used internally by the alert meeting service.
+@Component({
+  selector: 'cui-alert-meeting',
+  template: `
+    <ng-container *ngIf="avatar; else avatarTemplate" [ngTemplateOutlet]="avatar"></ng-container>
+    <div
+      [ngClass]="[
+        'cui-alert__content',
+        snoozeButtonVisible ? '' : 'cui-alert__content--wide'
+      ]"
```

Comment:
create a get property for this array, rather than using it directly in the template

Reply:
updated